Classes/Controller/AdminModuleController.php

```diff
@@ -112,21 +112,24 @@ protected function debugAction(
         ServerRequestInterface $request,
         ModuleTemplate $view,
     ): ResponseInterface {
-        $cmd = $request->getParsedBody()['tx_examples_admin_examples']['cmd'] ?? 'cookies';
-        switch ($cmd) {
-            case 'cookies':
-                $this->debugCookies();
-                break;
-            default:
-                // do something else
+        $body = $request->getParsedBody();
```

Comment:
The previous default was to fall to "cookies" if no action had been set, functionally this changes the logic workflow now. But I think that makes more sense to me, so all good, just making a note.

Reply:
After phpstan update this suddenly raised a warning

Reply:
getParsedBody() is allowed to return array|object|null - related to some psr/http-messgage interface update along with phpstan and having phpstan on a higher level. Needs proper is_array() check or dealing that it "may be a object" (json decoded body as stdClass etc).

We had simular occurances in one of our public extensions and already in the one or other project. That does not bubble up in the core due to the phpstan level 5 in core.